R/center.R

```diff
@@ -7,6 +7,10 @@
 #'  sequence of operations for this recipe.
 #' @param ... One or more selector functions to choose variables
 #'  for this step. See [selections()] for more details.
+#' @param case_weights A selector that specifies a single numeric column that
```

Comment:
Since we have the case weight classes and roles working, we won't use arguments for the case weights. They will automatically get picked up by the steps and used (or not) depending on the type of weight.

Reply:
Okay. It would make it cleaner to disallow the user from turning off case weights on a step by step basis.